calculator/operations.py

```diff
@@ -1,23 +1,37 @@
+import functools
+import operator
+from datetime import datetime
 
 def add(*args):
-    # your implementation here
-    pass
-
-def subtract(*args):
-    # your implementation here
-    pass
-
+    return sum(args)
+        
+subtract = lambda *args: args[0] - sum(args[1:])
+    
 def multiply(*args):
-    # your implementation here
-    pass
+    #product = 1
+    #for i in args[0]:
+        #product *= i
+    product = functools.reduce(lambda x, y: x*y, args)
+    return product
+    
 
 def divide(*args):
-    # your implementation here
-    pass
+    quotient = functools.reduce(lambda x,y: float(x)/y, args)
+    return quotient
+    
 
 def plot(*args):
     # OPTIONAL EXTRA CREDIT FUNCTION! 
     # See README for info.
     pass
 
+def create_new_calculator(operations={}):
```

Comment:
I don't think this is supposed to be here. `create_new_calculator` is handled by `main.py`